docs/sources/tutorials/get-started.md

```diff
@@ -71,17 +86,161 @@ services:
+
+## Configure {{< param "PRODUCT_NAME" >}}
+
+Once the local Grafana instance has been set up, the next step is to give detailed instructions to Alloy on which logs we want to scrape, how we want to process that data, and where we want the data sent. This is done through `config.alloy` file which contains components with instructions for Alloy to execute. Our configuration will connect components into a workflow.
+
+The examples run on a single host so that you can run them on your laptop or in a Virtual Machine. You are encouraged to try the examples using a `config.alloy` file and experiment with the examples yourself.
+
+For the following steps, create a file called `config.alloy` in your current working directory. 
+{{< param "PRODUCT_NAME" >}} has a feature that allows us to "hot reload" a configuration from a file. In a later 
+step, we will copy this file to where {{< param "PRODUCT_NAME" >}} will pick it up, and be able to reload without restarting 
+the system service.
+
+### First Component: Log files
+
+Put this component into the top of the `config.alloy` file:
+
+```alloy
+local.file_match "local_files" {
+    path_targets = [{"__path__" = "/var/log/*.log"}]
+    sync_period = "5s"
+}
+```
+
+In {{< param "PRODUCT_NAME" >}}'s configuration language, this creates a [local.file_match] component named `local_files` with an attribute that tells {{< param "PRODUCT_NAME" >}} which files to source, and to check every 5 seconds.
+
+### Second Component: Scraping
+
+Put this component next in the `config.alloy` file:
+
+```alloy
+loki.source.file "log_scrape" {
+    targets    = local.file_match.local_files.targets
+    forward_to = [loki.write.grafana_loki.receiver]
+    tail_from_end = true
+}
+```
+
+This configuration creates a [loki.source.file] component named `log_scrape`, and
+shows the pipeline concept of {{< param "PRODUCT_NAME" >}} in action:
+
+1. It applies to the `local_files` component (its "source" or target)
+2. It forwards the logs it scrapes to the "receiver" of another component called `grafana_loki` that we will define next
+3. It provides extra attributes and options, in this case, we will tail log files from the end and not ingest the entire
+past history
+
+### Third Component: Write Logs to Loki
+
+Place this component last in your configuration file:
+
+```alloy
+loki.write "grafana_loki" {
+  endpoint {
+    url = "http://localhost:3100/loki/api/v1/push"
+
+    // basic_auth {
+    //  username = "admin"
+    //  password = "admin"
+    // }
+  }
+}
+```
+
+We create a [loki.write] component named `grafana_loki` that points to `http://localhost:3100/loki/api/v1/push`. This completes a simple configuration pipeline.
+
+{{< admonition type="tip" >}}
+Notice that the `basic_auth` is commented out. Our local `docker-compose` stack does not require it; we include it in this example
+to show how you can configure auth for other environments. For further auth options, consult the [loki.write] component reference.
```

Comment:
```suggestion
The `basic_auth` is commented out because the local `docker-compose` stack doesn't require it. 
It is included in this example to show how you can configure authorization for other environments.
For further authorization options, refer to the [loki.write][] component reference.

[loki.write]: ../../reference/components/loki.write/
```